universal-login-sdk/lib/api/sdk.ts

```diff
@@ -95,6 +100,14 @@ class UniversalLoginSDK {
     this.deploymentObserver = this.deploymentObserver || new DeploymentObserver(this.blockchainService, this.config!.contractWhiteList);
   }
 
+  async getBalanceObserver(ensName: string) {
+    const walletContractAddress = await this.getWalletContractAddress(ensName);
+    ensureNotNull(walletContractAddress, WalletContractNotDeployed);
+    ensureNotNull(this.config, MissingConfiguration);
+
+    this.balanceObserver = this.balanceObserver || new BalanceObserver(this.balanceChecker, walletContractAddress, ACTIVE_TOKENS);
```

Comment:
return this.balanceObserver;

Reply:
Let's introduce 
```ts
interface SDKConfig {
  defaultPaymentOptions?: Message,
  observedTokens: []
}
```

By default take tokens from relayer SupportedTokens.

Reply:
Let's use TokenService to fetch tokens data.
TokenService should be moved from wallet to commons? (or sdk if hard)